File: problemas/iniciante/1177/1177.cs
using System;

class URI {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine());

        for(int i = 0; i < 1000; ++i){
            Console.WriteLine($"N[{i}] = {i % N}");
        }
    }
}

File: problemas/iniciante/1177/1177.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let mut numero: i32 = input.trim().parse().unwrap();

    for i in 0..1000 {
        println!("N[{}] = {}", i, i % numero);
    }
}
